src/lib.rs

```diff
@@ -396,4 +432,242 @@ mod test {
         assert_eq!(v.log_level(), Some(Level::Info));
         assert_eq!(v.log_level_filter(), LevelFilter::Info);
     }
+
+    #[test]
+    fn from_level_filter() {
+        // ErrorLevel
+        let v = Verbosity::<ErrorLevel>::from(LevelFilter::Off);
+        assert_eq!(v, Verbosity::new(0, 1));
```

Comment:
I think more important than what quiet/verbose levels are is to see that it produces the same `LevelFiltter`

btw for these tests, you can always iterate over those of the same T in a loop, with an array of inputs and expecteds.

Reply:
Subjectively (based on my experience), adding loops or conditionals to tests leads to worse tests. Tests in that form are as a general rule are more difficult to diagnose for failure. Often tests written this way don't take into account the main goal of testing, that tests should be useful when they fail. Specifically taking this approach here leads to tests that you have to interpret how the test is constructed and then which of the constructions led to failure. In one off situations that's fine, but as you add more and more of these types of tests then these little paper cuts build up. By taking this approach you also don't get to see all the failures, only the first failure in a potential list of failures. This makes fixing any errors when failure happens an O(N) rather than O(1) type problem.

I'm fine with making this change, but at the same time, I would ask whether it's worth your time to reconsider your position on using a tool like rstest for testing things which are naturally parameterized like this. You get the same result as a looped parameterized condition, but the complexity cost is amortized across all tests rather than paid by each test. I know that this may come across as belaboring a point (and apologize if it does come across as such). However the points you're bringing up about testing approach are solved problems with an existing tool, which when avoided means re-solving those same challenges manually. I also don't know what tradeoffs your considering with respect to not using rstest, so as mentioned I'm fine going with this approach (note the changes in https://github.com/clap-rs/clap-verbosity-flag/pull/115 at https://github.com/clap-rs/clap-verbosity-flag/pull/115/commits/7f435cde67459fc9a8c95c8bebd7a9808db39e0d

Reply:
I know very well the value of parameterized tests.  I deal with enough custom test frameworks / harnesses that I do not want to deal with any more at this time.  They are all a hack on what should be provided by the de facto test harness which is what I'm working to improve as part of my efforts with the T-testind-devex team in the Rust Project.

Reply:
Got it - makes sense. And agree definitely on should be part of the defaults :)